src/app/faq/faq.component.scss

```diff
@@ -0,0 +1,98 @@
+@import 'src/assets/styles/variables';
+
+.faq-container {
+  display: flex;
+  flex-direction: column;
+  background-image: url('/assets/images/faq-bg.png');
+  background-repeat: no-repeat;
+  height: 100%;
+  width: 100%;
+  &__header {
+    display: flex;
+    flex-direction: row;
+    margin-top: 48px;
+    &-title {
+      display: flex;
+      flex-direction: column;
+      margin-left: 26%;
+      @media screen and (max-width: 600px) {
+        margin-left: 10px;
+      }
+      &-icon {
+        color: $Cerise;
+        margin-left: 32%;
+        margin-top: 8%;
+        @media screen and (max-width: 600px) {
+          margin-left: 32%;
+        }
+      }
+      &-topic {
+        font-size: 48px;
+        margin: 0;
+      }
+    }
+    &-icon-size {
+      font-size: 45px;
+      padding: 16px;
+      @media screen and (max-width: 600px) {
+        font-size: 30px;
+        padding: 4px;
+      }
+    }
+  }
+  &-accordion {
+    display: block;
+    max-width: 60%;
+    margin-left: 16%;
+    margin-top: 38px;
+    @media screen and (max-width: 600px) {
+      max-width: 100%;
+      margin: 20px;
+    }
+  }
+}
+
+.example-accordion-item {
+  display: block;
+  border: solid 1px $gray;
+  margin-bottom: 16px;
+}
+
+.example-accordion-item-header {
+  display: flex;
+  align-content: center;
+  justify-content: space-between;
+}
+
+.example-accordion-item-description {
+  font-size: 0.85em;
+  color: $grayscale;
+}
+
+.example-accordion-item-header,
+.example-accordion-item-body {
+  padding: 16px;
+}
+
+.example-accordion-item-header:hover {
+  cursor: pointer;
+  background-color: $Cerise;
```

Comment:
please use camelCase for naming the variable

Reply:
resolved..................Thank you